packages/material-ui/src/Button/Button.js

```diff
@@ -365,7 +387,7 @@ Button.defaultProps = {
   color: 'default',
   component: 'button',
   disabled: false,
-  disableFocusRipple: false,
+  disableFocusRipple: true,
```

Comment:
Should we wait for a minor or can it be released in a patch? I would vote for a minor.

Reply:
Probably any choice would be wrong. It's a fix if you want strict material design but breaking if you rely on a focus ripple for some reason. Minor sounds like a good compromise